blueprints/addon/files/README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Compatibility
 
-* Ember.js v3.24 or above
+* Ember.js v3.28 or above
 * Ember CLI v3.24 or above
```

Comment:
```suggestion
* Ember CLI v3.28 or above
```

AFAIK, these can be in sync. Some more info can be found here: https://github.com/ember-cli/ember-cli/pull/9491.

Reply:
Makes sense, committed your change. I was used to see the CLI version be far behind ember-source in the past...